<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="OutliersTablelabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>802</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>  df 'Crime Scenes' : Column 'Area ID' Simple Outliers</string>
   </property>
  </widget>
  <widget class="QTableView" name="OutliersTableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>802</width>
     <height>425</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="StdDevDef">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>490</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Standard Deviation = 4555.666</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
